draft-romann-mud-constrained.md

```diff
@@ -72,7 +72,11 @@ Building upon the terminology defined in {{!RFC8520}}, this specification introd
 
 TODO. (Remove if there are no additional terms.)
 
-# Exposing a MUD URL using NDP
+# Exposing a MUD URL in Constrained Environments
```

Comment:
Isn't this basically the subject of the entire draft? I am unsure whether this heading might be redundant.

Reply:
I'd just remove the heading, other RFCs seem to be similarly structured, and nesting too many headings makes finding sections harder.